[webkitscmpy] Reduce autoinstalls on import

JonWBedard opened the pull request:
#### 7c8293e6d4d1879271b238a89607492c4acc6125
<pre>
[webkitscmpy] Reduce autoinstalls on import
<a href="https://bugs.webkit.org/show_bug.cgi?id=232574">https://bugs.webkit.org/show_bug.cgi?id=232574</a>
&lt;rdar://problem/84894275 &gt;

Reviewed by NOBODY (OOPS!).

* Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Remove whichcraft (webkitcorepy registers it).
* Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py: Remove unused webkitscmpy imports.
* Scripts/libraries/webkitscmpy/webkitscmpy/local/scm.py: Remote unused webkitcorepy import.
* Scripts/libraries/webkitscmpy/webkitscmpy/program/command.py: Move whichcraft to function.
* Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py: Move jinja2 to function.
* Scripts/libraries/webkitscmpy/webkitscmpy/remote/svn.py: Move fasteners, xmltodict import to functions.
</pre>

sundiamonde commented on Tools/Scripts/libraries/webkitscmpy/webkitscmpy/commit.py:
```diff
@@ -21,7 +21,6 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import json
-import six
```
insistence string types is a pretty basic need for python3 so I don't think moving the six imports is actually a win for clarity.  But six is also part of the system python packages so why are we auto installing it at all?

JonWBedard committed:
[webkitscmpy] Reduce autoinstalls on import
https://bugs.webkit.org/show_bug.cgi?id=232574
<rdar://problem/84894275>

Reviewed by NOBODY (OOPS!).

* Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Remove whichcraft (webkitcorepy registers it).
* Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py: Remove unused webkitscmpy imports.
* Scripts/libraries/webkitscmpy/webkitscmpy/local/scm.py: Remote unused webkitcorepy import.
* Scripts/libraries/webkitscmpy/webkitscmpy/program/command.py: Move whichcraft to function.
* Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py: Move jinja2 to function.
* Scripts/libraries/webkitscmpy/webkitscmpy/remote/svn.py: Move fasteners, xmltodict import to functions.

sundiamonde reviewed (commented):
nit: I probably would have added comments mentioning why we hid the imports, so some helpful soul doesn't come along and clean it up

I wonder if we should add something to check-webkit-style or somewhere that complains when we add new modules to the global namespace in webkitpy or webkitcmppy because that is the kind of thing that can slip past review, and I think we should need a very good reason to put new stuff in the global namespace
